### PR TITLE
INTYGFV-15592: Moved pu filtering and name search

### PR DESCRIPTION
**Pull request by mhornfeldt**

### CODE REVIEW

**Review by mikeyOne — APPROVED** [review]
Approved

File: web/src/main/java/se/inera/intyg/intygstjanst/web/service/impl/GetSickLeaveCertificatesImpl.java [hunk 1]
```diff
@@ -60,6 +73,10 @@ public List<SjukfallEnhet> get(String careProviderId, List<String> unitIds, List
```
Comment (mww-sopra-steria) [inline web/src/main/java/se/inera/intyg/intygstjanst/web/service/impl/GetSickLeaveCertificatesImpl.java:73]: Add sickLeaveLogMessageFactory.message(...) to log time to findAllSjukfallCertificate. Maybe call the log: GET_SICK_LEAVE_CERTIFICATES

**Review by mww-sopra-steria — APPROVED** [review]
See comment before merging!

**Review by mww-sopra-steria — CHANGES_REQUESTED** [review]
I forgot to check tests... You need to add some unit tests :-)

File: web/src/main/java/se/inera/intyg/intygstjanst/web/service/impl/GetSickLeaveCertificatesImpl.java (hunk 1 above)
Comment (mhornfeldt) [reply web/src/main/java/se/inera/intyg/intygstjanst/web/service/impl/GetSickLeaveCertificatesImpl.java:75]: Sorry I didn't see this comment before implementing a new message type. Is this OK?

**Pull request merged**